examples/dt/neutron/3cont_2comp_3net-ipv4-geneve-composable_provider_network

```diff
@@ -0,0 +1,66 @@
+# Deployed Topology 3cont_2comp_3net-ipv4-geneve-composable-vlan-provider-network
+
+## General information
+
+| Revision | Change              |  Date      |
+| -------: | :------------------ | :-----:    |
+|     v0.1 | Initial publication | 2024-01-16 |
+
+## Node topology
+
+| Node role                                  | bm/vm | amount |
+| ------------------------------------------ | ----- | ------ |
+| Openshift master/worker combo-node cluster | vm    | 3      |
+| Compute nodes                              | vm    | 2      |
+| Networker                                  | vm    | 3      |
+| CFME tester                                | vm    | 1      |
+
+
+## Services, enabled features and configurations
+
+| Service   | configuration                | Lock-in coverage?  |
+| --------- | ---------------------------- | ------------------ |
+| Neutron   | ML2/OVN, VLAN                | Must have          |
+|           | performance                  |                    |
```

Comment:
Performace tweaks for different services, uses these tripleo parameters https://github.com/redhat-openstack/infrared/blob/master/plugins/tripleo-overcloud/vars/overcloud/templates/performance.yml
Not sure if the job really requires them, but better to check with @eolivare, since it was added to d/s job config by his patch.
IMO, it would be enough if some of consolidated jobs will use the settings.

Reply:
According to @eolivare performance is not a 'must have' setting here

Reply:
fixed